src/styles/card.scss

```diff
@@ -0,0 +1,156 @@
+@import "variables";
+
+.card {
+  width: 200px;
+  height: 408px;
+  background-color: $background-color;
+  border-radius: 5px;
+  border: solid 1px $border-color;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  transition: all 300ms ease;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__title {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    line-height: 18px;
+    margin-top: 8px;
+    margin-left: 16px;
+    margin-right: 16px;
+    height: 36px;
+    width: 166px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 32px 19px;
+    background-image: url("../images/imac.jpeg");
```

Comment:
This should be an `img` tag, not a `background-image`. Because this image has semantic meaning, and we can't set `alt` on `background-image`